doc/source/User_guide/postprocessing.rst

```diff
@@ -1,33 +1,146 @@
+    hfss = Hfss()
+    hfss.analyze_nominal()
+
+    traces_to_plot = hfss.get_traces_for_plot(second_element_filter="P1*")
+    report = hfss.post.create_report(traces_to_plot)  # Creates a report in HFSS
+    solution = report.get_solution_data()
+    plt = solution.plot(solution.expressions)  # Matplotlib axes object.
+
+
+.. image:: ../Resources/sparams_w_matplotlib.jpg
+  :width: 800
+  :alt: S-Parameters report created with matplotlib
+
+
+PyAEDT can be used to plot any kind of report available in the Electronics Desktop interface.
+All available category can be accessed using ``reports_by_category`` class.
 
 .. code:: python
 
+    from pyaedt import Hfss
+    hfss = Hfss()
+    hfss.analyze_nominal()
+    # Create a 3d far field
+    new_report = hfss.post.reports_by_category.far_field(expressions="db(RealizedGainTotal)",
+                                                         setup_name=hfss.nominal_adaptive)
+
+
+Field plot can be plot directly in HFSS and can be exported to image files.
+
+.. code:: python
 
     from pyaedt import Hfss
     hfss = Hfss()
     hfss.analyze_nominal()
+
     cutlist = ["Global:XY"]
     setup_name = hfss.existing_analysis_sweeps[0]
     quantity_name = "ComplexMag_E"
     intrinsic = {"Freq": "5GHz", "Phase": "180deg"}
+    # Create a field plot
+    plot1 = hfss.post.create_fieldplot_cutplane(objlist=cutlist,
+                                                quantityName=quantity_name,
+                                                setup_name=setup_name,
+                                                intrinsincDict=intrinsic)
 
-    # create a field plot
-    plot1 = hfss.post.create_fieldplot_cutplane(cutlist, quantity_name, setup_name, intrinsic)
 
-    # create a 3d far field
-    new_report = hfss.post.reports_by_category.far_field("db(RealizedGainTotal)", hfss.nominal_adaptive)
+.. image:: ../Resources/field_plot.png
+  :width: 800
+  :alt: Post Processing features
 
-    # create a rectangular plot
-    report = hfss.post.reports_by_category.modal_solution("dB(S(1,1))")
-    report.create()
 
-    solutions = report.get_solution_data()
+PyAEDT leverages PyVista to export and plot fields outside AEDT, generating images and animations:
 
+.. code:: python
 
-.. image:: ../Resources/field_plot.png
+    from pyaedt import Hfss
+    hfss = Hfss()
+    hfss.analyze_nominal()
+    cutlist = ["Global:XY"]
+    setup_name = hfss.existing_analysis_sweeps[0]
+    quantity_name = "ComplexMag_E"
+    intrinsic = {"Freq": "5GHz", "Phase": "180deg"}
+    hfss.logger.info("Generating the image")
+    plot_obj = hfss.post.plot_field(
+            quantity="Mag_E",
+            objects_list=cutlist,
+            plot_type="CutPlane",
+            setup_name=setup_name,
+            intrinsics=intrinsic
+        )
+
+
+.. image:: ../Resources/pyvista_plot.jpg
   :width: 800
   :alt: Post Processing features
+
+
+PyAEDT includes a powerful class to generate pdf report that is based on python package ``fpdf2``.
```

Comment:
```suggestion
PyAEDT includes a powerful class to generate a PDF report that is based on the Python ``fpdf2`` package.
```
The code entity should have the noun that describes it following it rather than preceding it.